frontend/src/pages/EventSignInPage/index.tsx

```diff
@@ -41,15 +42,44 @@ function EventSignInPage(): JSX.Element {
               </Grid>
 
               <Grid item>
+                {
```

Comment:
The 2 <Typography> should be under a <Grid container> and this new <Grid container> should be under the <Grid item> on line 44

Something like:
```
<Grid item>
    <Grid component ...>
        <Typography>...</Typography>
        <Typography>...</Typography>
    </Grid>
</Grid>
```